sway-lsp/src/capabilities/code_actions/struct_decl/mod.rs

```diff
@@ -0,0 +1,25 @@
+pub(crate) mod struct_impl;
+pub(crate) mod struct_new;
+
+use sway_core::decl_engine::DeclId;
+use sway_types::Spanned;
+use tower_lsp::lsp_types::CodeActionOrCommand;
+
+use self::{struct_impl::StructImplCodeAction, struct_new::StructNewCodeAction};
+
+use super::{CodeActionContext, CodeActionTrait};
+
+pub(crate) fn code_actions(
+    decl_id: &DeclId,
+    ctx: CodeActionContext,
+) -> Option<Vec<CodeActionOrCommand>> {
+    let decl = ctx
+        .engines
+        .de()
+        .get_struct(decl_id.clone(), &decl_id.span())
+        .unwrap();
```

Comment:
nit: remove unwrap

Reply:
What should I use instead? The return type is `Result<ty::TyStructDeclaration, CompileError>`